index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>Portfolio</title>
+</head>
+<body>
+   
+        <!---Header-->
+        <header class="tool-bar">
+            <a href="#">Welcome</a>
+            <nav class="menu"><i class="fas fa-bars"></i></nav>
+        </header>
+           
+        <div class="container">
+            
+            <h2>
+                Hey There.<br>
+                I'm Somdotta
+            </h2>
+            <h4>I'm a Software Developer</h4>
+            <p>I can help you build a product , feature or<br>
+                 website Look through some of my work <br>
+                 and experience!
+                 If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <div class="links">
+                <i class="fab fa-angellist"></i>
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-github"></i>
+                <i class="fab fa-linkedin"></i>
+                <i class="fab fa-medium"></i>
+                  
+            </div>    
```

Comment:
The list of icons should be a ul tag to group them, and li per icon; each icon has a link (a tag).

Reply:
Thanks for the review.I'll do accordingly.